### ISSUE TITLE
Wyrestorm NHD-400 or NHD-000-CTL

### ISSUE THREAD

**Issue by dendryganarren** [bitfocus/companion-module-requests#653]
I would love to be able to send CEC-messsages through our Wyrecast system to switch on/off the TVs that we use for local magnification in Uppsala Cathedral.
A documented telnet-API is avaliable for the range of products named "NetworkHD"
https://wyrestorm0.sharepoint.com/PartnerResources/Shared%20Documents/WyreStorm%20Resources/Products/NetworkHD%20(New)/NetworkHD%20100-200-400-600-CTL/NetworkHD%20Documentation/NetworkHD_API_v6.2.pdf

**Comment by josephdadams** [comment]
Is this API available without a login?

**Comment by dendryganarren** [comment]
Yes, it's just Wyrestorm that's using sharepoint in a broken way - should have checked the link, sorry.
I've downloaded it and included it in this post

[NetworkHD_API_v6.3.pdf](https://github.com/bitfocus/companion-module-requests/files/7833276/NetworkHD_API_v6.3.pdf)
